EIPS/eip-6051.md

```diff
@@ -0,0 +1,464 @@
+---
+eip: 6051
+title: Private Key Encapsulation
+description: defines a specification for encapsulating private keys.
+author: Base Labs (@Base-Labs), Weiji Guo (@weiji-cryptonatty)
+discussions-to: https://ethereum-magicians.org/t/private-key-encapsulation-to-move-around-securely-without-entering-seed/11604
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-11-21
+---
+
+
+## Abstract
+
+This EIP proposes a mechanism to encapsulate a private key so that it could be securely relocated to another application without providing the seed. This EIP combines `ECIES` and optional signature verification under various choices to ensure that the private key is encapsulated to a known or trusted party.
```

Comment:
[Editorial] Suggest spell out `ECIES` what it's abbreviated for and provide relevant brief explanation

Reply:
added